Gemini/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "islandora/gemini",
+    "description": "ID Mapping Service between Fedora and Drupalf or Islandora",
+    "require": {
+        "silex/silex": "^2.0",
+        "doctrine/dbal": "~2.2"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.0",
+        "squizlabs/php_codesniffer": "^2.0",
+        "sebastian/phpcpd": "^3.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Islandora\\Gemini\\": "src/"
+        }
+    },
+    "license": "MIT",
+    "authors": [
+        {
```

Comment:
Yeah sure.  I think I copy/pasted this from before i did it there.  Hopefully next time I copy/paste a composer.json it'll be in there.